notebooks/picsure_2_harmonized_vars/main.py

```diff
@@ -0,0 +1,161 @@
+import os
+import callysto
+
+# Mock the notebook environment
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+
+def initial_setup_from_previous_notebook():
+    import os
+    import requests
+    url = 'https://raw.githubusercontent.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/master/NHLBI_BioData_Catalyst/python/python_lib/utils.py'
+    r = requests.get(url)
+    dir_name = os.path.join(os.path.dirname(__file__), 'python_lib')
+    os.makedirs(dir_name, exist_ok=True)
+    with open(f'{dir_name}/utils.py', 'wb') as f:
+        f.write(r.content)
+    with open(f'{dir_name}/__init__.py', 'wb') as f:
+        f.write(b'\n')
+    if not os.environ.get('PICSURE_TOKEN'):
+        raise RuntimeError('Please follow the instructions here to set PICSURE_TOKEN: https://terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20PIC-SURE%20API%20Python%20examples/notebooks/launch/Workspace_setup.ipynb')
+    with open('token.txt', 'w') as f:
+        f.write(os.environ.get('PICSURE_TOKEN'))
```

Comment:
Consider putting these files into a temporary directory, which will be cleaned up at end of execution:
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory